operator/molecule/requirements.txt

```diff
@@ -4,5 +4,7 @@ molecule
 openshift
 jmespath
 junit-xml
+kubernetes
 pyasn1>=0.4.6
 python-ldap
+docker
```

Comment:
Don't forget to make these same changes to our molecule Dockerfile - some of us don't have python running correctly on our local box and use the molecule docker image for running these tests :)

So add these to this [Dockerfile](https://github.com/kiali/kiali/blob/master/operator/molecule/docker/Dockerfile#L3) and test it by re-building the new docker image and run the maistra-e2e scenario:

```
MOLECULE_SCENARIO=maistra-e2e make molecule-docker-build molecule-test
```